org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/apt/model/RecordComponentElementImpl.java

```diff
@@ -42,7 +43,11 @@ public ExecutableElement getAccessor() {
 		ReferenceBinding binding = comp.declaringRecord;
 		if (binding instanceof SourceTypeBinding) {
 			MethodBinding accessor = ((SourceTypeBinding) binding).getRecordComponentAccessor(comp.name);
-			return new ExecutableElementImpl(_env, accessor);
+			return new ExecutableElementImpl(this._env, accessor);
```

Comment:
Looks like two "if" blocks are identical except first line, can that be extracted as ReferenceBinding line to avoid copy/paste the rest of the code?

Reply:
I have fixed it.